docs/apache-airflow/core-concepts/tasks.rst

```diff
@@ -149,82 +149,6 @@ is periodically executed and rescheduled until it succeeds.
         mode="reschedule",
     )
 
-If you merely want to be notified if a task runs over but still let it run to completion, you want :ref:`concepts:slas` instead.
-
-
-.. _concepts:slas:
-
-SLAs
```

Comment:
Maybe a sentence or two describing this feature as deprecated in 3.0? And that there are plans to add it back in 3.1?

Reply:
Added.